crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/tuple.py

```diff
@@ -66,3 +66,12 @@
 h1 = ((((1, 2))))
 h2 = ((((1, "qweiurpoiqwurepqiurpqirpuqoiwrupqoirupqoirupqoiurpqiorupwqiourpqurpqurpqurpqurpqurpqurüqurqpuriq"))))
 h3 = 1, "qweiurpoiqwurepqiurpqirpuqoiwrupqoirupqoirupqoiurpqiorupwqiourpqurpqurpqurpqurpqurpqurüqurqpuriq"
+
+i1 = ("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",)  # This should break
+
+# As of adding this fixture `black` replaces the non-breaking space with a space if followed by a space.
+# https://github.com/psf/black/blob/b4dca26c7d93f930bbd5a7b552807370b60d4298/src/black/comments.py#L122-L129
+i2 = ("",)  #         type: Add space before leading NBSP followed by spaces
+i3 = ("",)  #type: A space is added
+i4 = ("",)  #  type: Add space before leading NBSP followed by a space
+i5 = ("",)  # type: Add space before leading NBSP
```

Comment:
I was considering adding a dedicated module for comment fixtures. If that's preferred I can move this to one.

Reply:
I moved it to its own test file. It felt a bit arbitrary to have this in the tuple expression testing.

Reply:
Agreed. Thank you.